### ISSUE TITLE
Accessibility across the board

### ISSUE THREAD

**Issue by jackson-lewis** [jackson-lewis/shiftr#16]
Aware of a few issues where accessibility is being flagged by Lighthouse. 

- `aria-label` missing on logo anchor in header
- No `aria` controls to the primary navigation submenus
- No `aria` control of the `.header--offset-contents` element for mobile

**Comment by jackson-lewis** [comment]
Due to tight deadlines, it's going to be easier to include this update as part of v1.2
